src/devices_models/devices/common/energy_balance_constraint.jl

```diff
@@ -67,7 +67,7 @@ function energy_balance(
             psi_container.JuMPmodel,
             varenergy[name, t] ==
             varenergy[name, t - 1] + varin[name, t] * eff_in * fraction_of_hour -
-            (varout[name, t]) * fraction_of_hour / eff_out
+            (varout[name, t]) * fraction_of_hour * eff_out
```

Comment:
if we use this model, the `eff_out` values need to be larger than 1.0. If this is the case. It also needs to address the calculation of the these factors elsewhere in the code.

Reply:
You are correct. This should stay the way it was. 
In fact, the bug I tried to solve comes from the function call in storage.jl.